servatrice/src/servatrice.cpp

```diff
@@ -160,6 +160,14 @@ bool Servatrice::initServer()
         authenticationMethod = AuthenticationNone;
     }
 
+	QString maxUserLimitEnabled = settingsCache->value("security/enable_max_user_limit", "false").toString();
+	qDebug() << "Maximum user limit enabled: " << maxUserLimitEnabled;
+
+	if (maxUserLimitEnabled == "true"){
```

Comment:
Instead of using a string, you can use 
```
QString maxUserLimitEnabled = settingsCache->value("security/enable_max_user_limit", "false").toBool();
```
and then just ``` if (maxUserLimitEnabled) ```